BUILD

```diff
@@ -428,72 +428,83 @@ grpc_cc_library(
     ],
 )
 
+grpc_cc_library(
+    name = "support_manual_constructor",
```

Comment:
Do we also need to add rules for lib/support/memory.h and lib/support/vector.h?

Might be worth doing an exhaustive check for all C++ code in the directory, to make sure we've got rules for all of them.

Also, do we need equivalents for these rules in build.yaml?